app/components/post_list/post_list.tsx

```diff
@@ -65,8 +65,8 @@ type ScrollIndexFailed = {
 };
 
 const CONTENT_OFFSET_THRESHOLD = 160;
+const SCROLL_EVENT_THROTTLE = Platform.select({android: 17, default: 60});
```

Comment:
The actual fix. This value is being fed into scrollEventThrottle in the FlatList.

iOS at 60 is not showing any issue (on simulator), but android at even 20 is causing the y value to not register the value of 0 even though the FlatList has been scrollToOffset of 0.  This is because the scrollEventThrottle is throttling and would miss some of the event.  At the value of 0-16, it will no longer throttling.  I was able to use 17 and having it still throttling (maybe?) but not losing the event when it hit 0.